src/pages/Auth/SignIn/SignIn.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SignInForm from './SigninForm';
+
+const SignIn = () => (
```

Comment:
- [ ] When I try to sign in, I need to press the "log in" button twice for it to work:

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/c05fb035-3eac-4511-b40b-d603ca06ca5e)

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/ae2b7bb4-11f9-48e2-bcb5-f97d7a59353f)

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/cf70b07c-62e0-4acd-9eec-f541b6c83a4a)

On second press, the button works.  Kindly make it so the log in feature works on first try. 👍